PostBy/View Controllers/SettingsViewController.m

```diff
@@ -44,6 +55,41 @@ - (void)viewDidLoad {
     PFFileObject *profilePicObj = [PFUser.currentUser valueForKey:@"profilePicture"];;
     NSURL *url = [NSURL URLWithString:profilePicObj.url];
     [self.profilePicture setImageWithURL:url];
+    
+    self.INVALID_CHOICE = -1;
+    self.DELETE_CHOICE = self.INVALID_CHOICE;
+    [self createDropDown];
+}
+
+- (void)createDropDown {
+    CGFloat x = (CGRectGetWidth(self.view.frame)-240)/2;
+    CGFloat y = 300;
+    CGFloat width = 240;
+    CGFloat height = 37;
+    CGRect frame = CGRectMake(x, y, width, height);
+    ManaDropDownMenu *menu = [[ManaDropDownMenu alloc] initWithFrame:frame title:@"Choose data to delete"];
+    menu.delegate = self;
+    menu.textOfRows = @[@"Likes", @"Dislikes", @"Posts", @"Comments", @"Account"];
+    menu.numberOfRows = menu.textOfRows.count;
+    
+    menu.activeColor = [UIColor redColor];
+    menu.inactiveColor = [UIColor redColor];
+    // Super light gray background
+    menu.titleViewColor = [UIColor colorWithRed:(250/255.0) green:(250/255.0) blue:(250/255.0) alpha:1];
+    
+    // Indexes based on textOfRows
+    self.INVALID_CHOICE = -1;
```

Comment:
Not sure whether the language supports enum structure. 

Enumeration can store all the options with the same data type in a single storage unit of that type. It can be a good way for heap space saving.

Reply:
Implemented enum! Will post the commit soon.